src/screens/StatisticsScreen/index.js

```diff
@@ -1,9 +1,13 @@
+// @flow
+
 import { connect, } from 'react-redux'
 import { StatisticsScreen, } from './StatisticsScreen'
+import type { RootState, } from '../../flowAliases'
+import type { Props, } from './StatisticsScreen'
```

Comment:
That's why `Props` is a bad idea for naming. It is not clear which type you export